src/fu-main.c

```diff
@@ -516,7 +516,7 @@ fu_main_get_device_family (FuMainAuthHelper *helper, GError **error)
 
 	/* device itself */
 	devices_possible = g_ptr_array_new_with_free_func ((GDestroyNotify) g_object_unref);
-	g_ptr_array_add (devices_possible, g_object_ref (device));
+	g_ptr_array_add (devices_possible, g_steal_pointer (&device));
```

Comment:
NAK, else `sibling == device` doesn't work as device is NULL. With this hunk removed, LGTM.

Reply:
OK thanks, wasn't sure about this one - and it worked both ways.  I'll remove and merge.